File: aws/lambda/python/compare-yourself/cy-store-data.py
import json
import boto3

dynamodb = boto3.client(
    'dynamodb', region_name='sa-east-1', api_version='2012-08-10')


def lambda_handler(event, context):
    params = {
        'Item': {
            'Userid': {
                'S': event['userId']
            },
            'Age': {
                'N': event['age']
            },
            'Height': {
                'N': event['height']
            },
            'Income': {
                'N': event['income']
            }
        },
        'TableName': 'compare-yourself'
    }
    response = dynamodb.put_item(**params)
    print(response)
    return response

File: aws/lambda/nodejs/compare-yourself/cy-store-data.js
const AWS = require("aws-sdk");
const dynamodb = new AWS.DynamoDB({
  region: "sa-east-1",
  apiVersion: "2012-08-10"
});

exports.handler = (event, context, callback) => {
  const params = {
    Item: {
      Userid: {
        S: event.userId
      },
      Age: {
        N: event.age
      },
      Height: {
        N: event.height
      },
      Income: {
        N: event.income
      }
    },
    TableName: "compare-yourself"
  };

  dynamodb.putItem(params, function(err, data) {
    if (err) {
      console.log(err);
      callback(err);
    } else {
      console.log(data);
      callback(null, data);
    }
  });
};
